aspnetcore/mvc/controllers/filters.md

```diff
@@ -134,23 +137,48 @@ Here's an example that illustrates the order in which filter methods are called
 This sequence shows:
 
 * The method filter is nested within the controller filter.
-* The controller filter is nested within the global filter. 
+* The controller filter is nested within the global filter.
 
-To put it another way, if you're inside an async filter's On*Stage*ExecutionAsync method, all of the filters with a tighter scope run while your code is on the stack.
+### Controller and Razor Page level filters
 
-> [!NOTE]
-> Every controller that inherits from the `Controller` base class includes `OnActionExecuting` and `OnActionExecuted` methods. These methods wrap the filters that run for a given action:  `OnActionExecuting` is called before any of the filters, and `OnActionExecuted` is called after all of the filters.
+Every controller that inherits from the <xref:Microsoft.AspNetCore.Mvc.Controller> base class includes[Controller.OnActionExecuting](xref:Microsoft.AspNetCore.Mvc.Controller.OnActionExecuting*),  [Controller.OnActionExecutionAsync](xref:Microsoft.AspNetCore.Mvc.Controller.OnActionExecutionAsync*)[Controller.OnActionExecuted](xref:Microsoft.AspNetCore.Mvc.Controller.OnActionExecuted*)
+`OnActionExecuted` methods. These methods:
+
+* Wrap the filters that run for a given action.
```

Comment:
These methods only wrap the action filters and not e.g. resource filters that run for an action. I don't know if there's a need to be explicit about that, but I'm just pointing it out in case it's not already clear.